### ISSUE TITLE
Frontend issues 

### ISSUE THREAD

**Issue by PAVAN-VANAM** [divyansh-2005/my-calendar-app#5]
Hello @divyansh-2005 

I found a few issues in the Frontend Application.

**Describe the issues**
1 .  There is not a proper flow of the website. We want a login page we need to go by typing /login and /register
2.  There is no menu component and a user-profile view component 
3.  Also, I noticed that there is no landing page.  We have to create a landing page and give access to login and register pages after we navigate to the Home page 
4. Also, I think we need to improve styles to make attractive pages using tailwind-CSS or CSS

I would like to work on the above issues Can U assign me?

**Comment by divyansh-2005** [comment]
@PAVAN-VANAM work on this, share screenshot/video of proposed change

**Comment by kottarivaibhav** [comment]
Yeah , work on css , once my pull request is accepted , I have already installed tailwind css in the repo . you can move on with that .
